CONTRIBUTING.md

```diff
@@ -49,6 +49,35 @@ When you send a PR, just make sure that:
 * You make the PR on the same branch you based your changes on. If you see commits
 that you did not make in your PR, you're doing it wrong.
 * Also don't forget to add a comment when you update a PR with a ping to the maintainer (``@username``), so he/she will get a notification.
+* Squash your commits into one commit. (see the next chapter)
+
+## Squash your commits
+
+If you have 3 commits. So start with:
+
+```bash
+git rebase -i HEAD~3
+```
+
+An editor will be opened with your 3 commits, all prefixed by `pick`.
+
+Replace all `pick` prefixes by `fixup` (or `f`) **except the first commit** of the list.
+
+Save and quit the editor.
+
+After that, all your commits where squashed into the first one and the commit message of the first commit.
+
+If you would like to rename your commit message type:
+
+```bash
+git commit --amend
```

Comment:
Well, this method works, but you can do all in the same type.

On rebase, replace `pick` by `reword` on the commit you want to rewrite and an another editor will appear to edit the comment.

But I don't know. This method looks better for newbies.